manifests/host.pp

```diff
@@ -7,6 +7,9 @@
   Hash $options     = {},
   String $comment   ='',
   Boolean $ignored  = false,
+  Optional[Array[String]] $on_commit = undef,
```

Comment:
Please align the `=` and enforce the minimal string length:
```suggestion
  Optional[Array[String[1]]] $on_commit = undef,
```

Reply:
Good call on this one.  Update coming...